modules/ROOT/partials/import-database.adoc

```diff
@@ -28,6 +28,7 @@ See <<_public_traffic>> below for more information.
 
 `database upload` is a `neo4j-admin` command that you can run to upload the contents of a Neo4j database into an Aura instance, regardless of the database's size.
 Keep in mind that the database you want to upload may run a different version of Neo4j than your Aura instance.
+Additionally, the machine you run the `database upload` command from must have access to the Aura instance you want to upload the database to.
```

Comment:
```suggestion
Additionally, the machine from which you run the `database upload` command must have access to the Aura instance to which you want to upload the database.
```
or similar to the Operation manual:
```
Additionally, your Neo4j Aura instance must be accessible from the machine running `neo4j-admin`. 
Otherwise, the upload will fail with SSL errors.
```